src/App.js

```diff
@@ -15,49 +15,46 @@ import SearchForm from './components/SearchForm';
 import ReleaseDateToggle from './components/ReleaseDateToggle';
 import MyModal from './components/Modal';
 import Form from './components/Form';
```

Comment:
create pleas index file fo export in ./components

Reply:
fix